packages/core/core/src/PackagerRunner.js

```diff
@@ -67,6 +68,14 @@ export default class PackagerRunner {
       bundle
     });
 
+    let depToBundlePath = generateDepToBundlePath(internalBundle);
+
+    // Inject the dependency-to-relative-path mapping into all the bundle's
+    // assets so that substitution can occur when the asset's output is generated.
+    internalBundle.assetGraph.traverseAssets(asset => {
+      asset.depToBundlePath = depToBundlePath;
```

Comment:
Wonder if we should just do the replacement after the packager runs instead of adding this to individual assets?

Reply:
I agree -- this feels cleaner. We'll still have to tackle the sourcemaps issue post-packager, but I don't think it'll be any easier/more difficult.